client/blocks/app-promo/index.tsx

```diff
@@ -50,17 +49,13 @@ export const AppPromo = ( {
 				alt="WordPress and Jetpack app"
 			/>
 			<div className="app-promo__title">
-				<CardHeading tagName="h2">
-					{ preventWidows( title || translate( 'Get our mobile app' ) ) }
-				</CardHeading>
-				{ ! showBadge && <h3 className="app-promo__subheader">{ preventWidows( subheader ) }</h3> }
+				<CardHeading tagName="h2">{ title || translate( 'Get our mobile app' ) }</CardHeading>
+				{ ! showBadge && <h3 className="app-promo__subheader">{ subheader }</h3> }
 			</div>
 
 			{ showBadge && (
 				<div className="app-promo__app-badges">
-					{ subheader && (
-						<p className="app-promo__app-badges-text">{ preventWidows( subheader ) }</p>
```

Comment:
`AppPromo`, used in home screen and discover section of reader:

<img width="753" alt="Screenshot 2025-05-06 at 4 25 23 pm" src="https://github.com/user-attachments/assets/252432e1-76a6-499a-bb64-9398f150682c" />